cache/redisOps.js

```diff
@@ -422,6 +422,29 @@ module.exports = {
     return redisClient.batch(cmds).execAsync();
   },
 
+  /**
+   * Get samples for a given aspect
+   * @param  {String} aspName - Aspect Name
+   * @returns {Promise} - Resolves to array of samples
+   */
+  getSamplesFromAspectName(aspName) {
```

Comment:
@iamigo I wanted to create a test for this function, but I do not have aspSubjMap in redis to test this. Looks like we are using an independent utility [aspectSubjectMap](https://github.com/salesforce/refocus-utilities/tree/master/src/sampleStore/aspectSubjectMap) to create this set in redis. Any way to work around this?

Reply:
Looks like we're implementing this out of order--that separate utility is to populate the aspsubmap from scratch when there is none, but we do need to update all the places where we will be keeping that map up to date: aspect unpublish, aspect delete, sample post, sample put, sample patch, sample upsert, sample bulk upsert, sample delete. If that doesn't exist, @pallavi2209, can you check for an existing story for it, and if one does not exist, create it?